### ISSUE TITLE
Add a start screen for game

### ISSUE THREAD

**Issue by PrajwalKrishna** [PK1210/SuperMario#2]
The game does not have a starting screen as of now. I wish that the game has a start screen where controls would be shown rest depends on creativity of person implementing.

**Comment by RodolfoFerro** [comment]
Hi @PK1210 is there a CONTRIBUTING.md file for this repo?

**Comment by zeroxenator** [comment]
Hey, Can I take this?

**Comment by PrajwalKrishna** [comment]
Yes sure you can take this alas no I don't have a CONTRIBUTING.md for this repo. But I believe that you can go ahead with the Start screen/
